app/src/lib/stores/git-store.ts

```diff
@@ -949,9 +949,14 @@ export class GitStore extends BaseStore {
           currentBranch.upstream
         )
         this._aheadBehind = await getAheadBehind(this.repository, range)
-        this.emitUpdate()
+      } else {
+        this._aheadBehind = null
       }
+    } else {
+      this._aheadBehind = null
     }
+
+    this.emitUpdate()
```

Comment:
We haven't run into any problems with not resetting `aheadBehind` in fetch before since all fetches are followed by a `status` call which itself refreshes (and resets) the `aheadBehind` but now that we're running fetch stand-alone it makes sense for us to be more diligent here.